Dockerfile

```
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.17.7 as builder
+From registry.access.redhat.com/ubi8/go-toolset:1.18.4-20 as builder
```

There is a `1.18` floating tag for the `ubi8/go-toolset` image. Using it would avoid having to bump the patch/build version explicitly.

```suggestion
From registry.access.redhat.com/ubi8/go-toolset:1.18 as builder
```